rcon/player_history.py

```diff
@@ -227,6 +231,31 @@ def ban_if_blacklisted(rcon, steam_id_64, name):
             )
 
 
+def ban_if_has_vac_bans(rcon, steam_id_64, name):
+    try:
+        autoban_threshold = int(AUTOBAN_TRIGGER_THRESHOLD)
+    except ValueError: # No (proper) value is given
+        return
```

Comment:
If you go for the 0 as default then you could log that exception. That'd avoid some awkward trouble shooting sessions in the tech support channel :)